src/Behat/Gherkin/Gherkin.php

```diff
@@ -82,16 +82,16 @@ public function setBasePath($path)
     /**
      * Loads & filters resource with added loaders.
      *
-     * @param mixed                    $resource Resource to load
-     * @param FeatureFilterInterface[] $filters  Additional filters
+     * @param mixed                        $resource Resource to load
+     * @param list<FeatureFilterInterface> $filters  Additional filters
```

Comment:
the implementation does not care about keys, so there is no need to enforce them being compatible with a list. `T[]` and `list<T>` are **not** the same type (`T[]` means `array<T>`, i.e. `array<int|string, T>`)